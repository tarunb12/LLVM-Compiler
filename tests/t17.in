// Function test (combinations)

int choose(int n, int k) {
    if (k == 0 || k == n) {
        return 1
    }
    return choose(n - 1, k - 1) + choose(n - 1, k)
}

int main() {
    int n = 16
    int k = 5
    printf("C(%d, %d) = %d", n, k, choose(n, k))
}