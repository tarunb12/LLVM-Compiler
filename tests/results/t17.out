C(16, 5) = 4368